testsuite/cluster-test/src/cluster_swarm/job_template.yaml

```diff
@@ -19,11 +19,20 @@ spec:
       containers:
       - name: main
         image: {image}
+        volumeMounts:
```

Comment:
may I ask what is this file used for and when do we need to touch it?

Reply:
job_template is used when `Instance::util_cmd` is called to run arbitrary command on validator node.
This just adds ability for whatever command runs this util to access validator database and configs